tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -148,73 +148,77 @@ HloInstruction *PadOperandToMultipleOf16(absl::Span<const int64_t> batch_dims,
   return PadOperandToTargetShape(padded_shape, x);
 }
 
-// Recursively collects unary, divide, dynamic-slice, pad or multiply operands
-// of instr until an instruction with FP8 element type is reached. Returns
-// std::nullopt when no FP8 instruction is reached.
-std::optional<std::vector<HloInstruction *>> FindF8SubgraphRecursive(
+// Recursively collects unary, divide, dynamic-slice, pad, multiply or select
+// operands of instr until an instruction with FP8 element type is reached.
+// Returns an empty vector when no FP8 instruction is reached.
```

Comment:
Comment what the int is. That is, the index of the operand of the next op in the chain.